src/index.ts

```diff
@@ -162,32 +164,30 @@ var formatTypeMap: {[type: string]: 'number' | 'time'} = {
 */
 function supplementOptions(options: Option, vlSpec: TopLevelExtendedSpec) {
   // fields to be supplemented by vlSpec
-  var supplementedFields: FieldOption[] = [];
+  let supplementedFields: FieldOption[] = [];
```

Comment:
const?

Try to replace all let with const and then only change the ones that need to be changed.